static/app/utils/performance/suspectSpans/suspectSpansQuery.tsx

```diff
@@ -10,7 +11,8 @@ import withApi from 'app/utils/withApi';
 import {SuspectSpans} from './types';
 
 type SuspectSpansProps = {
-  spanOps: string[];
+  perSuspect?: number;
```

Comment:
Not that important but... I know I approved it in the other PR but looking at it now `perSuspect` is a bit hard to interpret as the perPage? 🤔